src/connector/src/schema/protobuf.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::{BTreeMap, HashMap};
+
+use prost_reflect::MessageDescriptor;
+
+use crate::parser::{EncodingProperties, ProtobufParserConfig, ProtobufProperties};
+
+pub async fn fetch_descriptor(
+    _with_properties: &HashMap<String, String>,
+    format_options: &BTreeMap<String, String>,
+) -> Result<MessageDescriptor, &'static str> {
```

Comment:
On the source side:
https://github.com/risingwavelabs/risingwave/blob/58e6f9d608694c9ad01ede024194d5d1a3acf5d1/src/frontend/src/handler/create_source.rs#L333
https://github.com/risingwavelabs/risingwave/blob/58e6f9d608694c9ad01ede024194d5d1a3acf5d1/src/connector/src/parser/protobuf/parser.rs#L80
https://github.com/risingwavelabs/risingwave/blob/58e6f9d608694c9ad01ede024194d5d1a3acf5d1/src/connector/src/parser/avro/parser.rs#L115